devel/ansible/roles/dev/files/motd

```diff
@@ -11,6 +11,7 @@ bshell:      Get a handy python shell initialized with Bodhi models.
 bstart:      Start the Bodhi service.
 bstop:       Stop the Bodhi service.
 btest:       Run Bodhi's test suite.
+btest3:      Run Bodhi's test suite with Python 3.
```

Comment:
We could make btest run Python 3 and just rely on CI for Python 2 if you like. This is also fine.